src/pages/home/sidebar/SidebarScreen/index.js

```diff
@@ -42,7 +46,9 @@ function SidebarScreen(props) {
             <BaseSidebarScreen
                 // eslint-disable-next-line react/jsx-props-no-spreading
                 {...props}
+                shouldShowOfflineIndicator={false}
             >
+                {props.isSmallScreenWidth && props.network.isOffline && <OfflineIndicator style={[StyleUtils.getBackgroundColorStyle(themeColors.greenHighlightBackground)]} />}
```

Comment:
It seems the offline indicator is unrelated to this PR's purpose. Can you clarify if this is intentional?

Reply:
Sure - the `coverScreen` prop set to false in `react-native-modal` changes the component from `Modal` to `View`. It caused the `OfflineIndicator` component to become visible, because it was adjacent to the `PopoverMenu`. I decided to slightly change the components' order to make sure that everything appears correctly on the screen, and the `PopoverMenu` stays on top.

Reply:
Unfortunately I was checking it, while I was writing this code, and a simple `zIndex` change will not work. The problem is most probably connected to the component tree and their `position` value, so when I was playing with it, it seemed that some bigger refactor would have to be applied in order to make it work with `zIndex`.